third_party/dml/ci/tests/tfdml_test_runner.py

```diff
@@ -73,13 +73,18 @@ def _run_test(
 
   env_copy = _get_tf_env(exe_path, test_framework)
 
+
   if log_device_placement:
     env_copy["TF_CPP_LOG_DEVICE_PLACEMENT"] = "1"
 
   if os.name != "nt":
     env_copy["TEST_SRCDIR"] = exe_path + ".runfiles"
 
   if test_framework == "abseil":
+    # Specifying a port server for portpicker is necessary to avoid race
+    # conditions when searching for unused ports. @unittest-portserver is the
+    # sample port server that portpicker provides for test environments.
+    env_copy["PORTSERVER_ADDRESS"] = "@unittest-portserver"
```

Comment:
I'm not familiar with portpicker, but I assume it starts this server automatically?

Reply:
The portserver will start the server with `@unittest-portserver` by default, and this environment variable tells portpicker which address to listen to.